Documentation/guides/LayoutCodeBehind.md

```diff
@@ -337,23 +360,48 @@ namespace Example
       }
     }
 
-    public override void SetContentView (global::Android.Views.View view, global::Android.Views.ViewGroup.LayoutParams @params)
-    {
+    void SetContentView (global::Android.Views.View view, global::Xamarin.Android.Design.LayoutBinding.OnLayoutItemNotFoundHandler onLayoutItemNotFound) {
+      __layout_binding = new global::Binding.Main (view, onLayoutItemNotFound);
+      bool callBase = true;
+      OnSetContentView (view, ref callBase);
+      if (callBase) {
+        base.SetContentView (view);
+      }
+    }
+
+    public override void SetContentView (global::Android.Views.View view, global::Android.Views.ViewGroup.LayoutParams @params) {
```

Comment:
Given that we're generating all this code, why not use default parameters here & elsewhere?

```csharp
public override void SetContentView (
    global::Android.Views.View view,
    global::Android.Views.ViewGroup.LayoutParams @params,
    global::Xamarin.Android.Design.LayoutBinding.OnLayoutItemNotFoundHandler onLayoutItemNotFound = null)
{
  ...
}
```